[2020-07-14 12:51:14,648] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: datapipeline.stitching 2020-07-14T04:41:33.757061+00:00 [queued]>
[2020-07-14 12:51:14,658] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: datapipeline.stitching 2020-07-14T04:41:33.757061+00:00 [queued]>
[2020-07-14 12:51:14,658] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-07-14 12:51:14,658] {taskinstance.py:880} INFO - Starting attempt 1 of 4
[2020-07-14 12:51:14,658] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-07-14 12:51:14,666] {taskinstance.py:900} INFO - Executing <Task(BashOperator): stitching> on 2020-07-14T04:41:33.757061+00:00
[2020-07-14 12:51:14,670] {standard_task_runner.py:53} INFO - Started process 27916 to run task
[2020-07-14 12:51:14,714] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: datapipeline.stitching 2020-07-14T04:41:33.757061+00:00 [running]> tony-ThinkPad-X1-Carbon-7th
[2020-07-14 12:51:14,722] {bash_operator.py:82} INFO - Tmp dir root location: 
 /tmp
[2020-07-14 12:51:14,723] {bash_operator.py:105} INFO - Temporary script location: /tmp/airflowtmpi1hfqfze/stitchingm6s03see
[2020-07-14 12:51:14,723] {bash_operator.py:115} INFO - Running command: python /home/tony/opencv/stitching.py --video l1.mp4 r1.mp4 --stop_frame 10
[2020-07-14 12:51:14,728] {bash_operator.py:122} INFO - Output:
[2020-07-14 12:51:15,727] {bash_operator.py:126} INFO - 
[2020-07-14 12:51:15,727] {bash_operator.py:126} INFO - (python:27918): GStreamer-CRITICAL **: 12:51:15.727: gst_element_make_from_uri: assertion 'gst_uri_is_valid (uri)' failed
[2020-07-14 12:51:15,727] {bash_operator.py:126} INFO - [ WARN:0] global /home/tony/opencv/modules/videoio/src/cap_gstreamer.cpp (713) open OpenCV | GStreamer warning: Error opening bin: no element "input_vid"
[2020-07-14 12:51:15,727] {bash_operator.py:126} INFO - [ WARN:0] global /home/tony/opencv/modules/videoio/src/cap_gstreamer.cpp (480) isPipelinePlaying OpenCV | GStreamer warning: GStreamer: pipeline have not been created
[2020-07-14 12:51:15,728] {bash_operator.py:126} INFO - 
[2020-07-14 12:51:15,729] {bash_operator.py:126} INFO - (python:27918): GStreamer-CRITICAL **: 12:51:15.728: gst_element_make_from_uri: assertion 'gst_uri_is_valid (uri)' failed
[2020-07-14 12:51:15,729] {bash_operator.py:126} INFO - [ WARN:0] global /home/tony/opencv/modules/videoio/src/cap_gstreamer.cpp (713) open OpenCV | GStreamer warning: Error opening bin: no element "input_vid"
[2020-07-14 12:51:15,729] {bash_operator.py:126} INFO - [ WARN:0] global /home/tony/opencv/modules/videoio/src/cap_gstreamer.cpp (480) isPipelinePlaying OpenCV | GStreamer warning: GStreamer: pipeline have not been created
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO - 
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO - Stitching sample
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO - ================
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO - 
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO - Show how to use Stitcher API from python in a simple way to stitch panoramas
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO - or scans.
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO - 
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO - usage: stitching.py [-h] [--mode {0,1}] [--output OUTPUT]
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO -                     [--img IMG [IMG ...]] [--videos VIDEOS [VIDEOS ...]]
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO -                     [--stop_frame STOP_FRAME] [--view]
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO - 
[2020-07-14 12:51:15,738] {bash_operator.py:126} INFO - Stitching sample.
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO - 
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO - optional arguments:
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -   -h, --help            show this help message and exit
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -   --mode {0,1}          Determines configuration of stitcher. The default is
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -                         `PANORAMA` (0), mode suitable for creating photo
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -                         panoramas. Option `SCANS` (1) is suitable for
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -                         stitching materials under affine transformation, such
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -                         as scans.
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -   --output OUTPUT       Resulting image. The default is `result.jpg`.
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -   --img IMG [IMG ...]   input images
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -   --videos VIDEOS [VIDEOS ...]
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -                         input videos
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -   --stop_frame STOP_FRAME
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -                         limit of frames to stitch
[2020-07-14 12:51:15,739] {bash_operator.py:126} INFO -   --view                view stitch
[2020-07-14 12:51:15,740] {bash_operator.py:126} INFO - 
[2020-07-14 12:51:15,740] {bash_operator.py:126} INFO - ['l1.mp4', 'r1.mp4']
[2020-07-14 12:51:15,740] {bash_operator.py:126} INFO - can't read image
[2020-07-14 12:51:15,893] {bash_operator.py:130} INFO - Command exited with return code 255
[2020-07-14 12:51:15,903] {taskinstance.py:1145} ERROR - Bash command failed
Traceback (most recent call last):
  File "/home/tony/.virtualenvs/cv/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 983, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/tony/.virtualenvs/cv/lib/python3.6/site-packages/airflow/operators/bash_operator.py", line 134, in execute
    raise AirflowException("Bash command failed")
airflow.exceptions.AirflowException: Bash command failed
[2020-07-14 12:51:15,905] {taskinstance.py:1168} INFO - Marking task as UP_FOR_RETRY
[2020-07-14 12:51:24,645] {logging_mixin.py:112} INFO - [2020-07-14 12:51:24,645] {local_task_job.py:103} INFO - Task exited with return code 1
